src/components/panel/panel.spec.js

```diff
@@ -117,6 +120,110 @@ describe('$mdPanel', function() {
     expect(parentEl.childElementCount).toEqual(0);
   });
 
+  describe('component logic: ', function() {
+    it('should allow templateUrl to specify content',
+        inject(function($templateCache) {
```

Comment:
Rather than inject `$templateCache` for this one test, can you move it to a file-level var? I prefer keeping `inject` out of all individual tests so that later, when someone else adds a test, you don't end up with 30 different tests that all` inject` the same thing.